Formula/linux-headers.rb

```diff
@@ -1,13 +1,22 @@
 class LinuxHeaders < Formula
   desc "Header files of the Linux kernel"
   homepage "https://kernel.org/"
-  url "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.4.80.tar.gz"
-  sha256 "291d844619b5e7c43bd5aa0b2c286274fc5ffe31494ba475f167a21157e88186"
+  url "https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.9.16.tar.gz"
+  sha256 "f4e27357aba6d5a69ab473e53a93353f9bed1a7e8ee43a6d8e186770d6a79586"
   license "GPL-2.0-only"
 
-  depends_on :linux
+  on_macos do
+    depends_on "gnu-sed" => :build
+
+    # skip building archscripts which have host system dependencies
+    patch :DATA
```

Comment:
Has this patch been submitted upstream? We don't usually accept patches that haven't.

Reply:
this is a workaround rather than a 'real' patch. there are probably good reasons why if you're compiling the whole kernel you might need this, but given this is just to get the headers it's easier just to skip that step.

Reply:
Can't you submit this as a patch upstream to allow skipping it with a `./configure` flag or whatever?